doc/articles/features/using-skia-gtk.md

```diff
@@ -57,9 +57,52 @@ If you want to upgrade **SkiaSharp** to a later version, you'll need to specify
 
 ```xml
 <ItemGroup>
-   <PackagReference Include="SkiaSharp" Version="2.88.0" />
-   <PackagReference Include="SkiaSharp.Harfbuzz" Version="2.88.0" />
-   <PackagReference Include="SkiaSharp.NativeAssets.Linux" Version="2.88.0" />
-   <PackageReference Update="SkiaSharp.NativeAssets.macOS" Version="2.88.0" />
+   <PackagReference Include="SkiaSharp" Version="2.88.3" />
+   <PackagReference Include="SkiaSharp.Harfbuzz" Version="2.88.3" />
+   <PackagReference Include="SkiaSharp.NativeAssets.Linux" Version="2.88.3" />
+   <PackageReference Update="SkiaSharp.NativeAssets.macOS" Version="2.88.3" />
 </ItemGroup>
 ```
+
+### .NET Native AOT support
+
+Building an Uno Platform Skia+GTK app with .NET (7+) Native AOT requires, GtkSharp 3.24.24.38 (or later), or Uno Platform 4.7 (or later).
+
+To build an app with this feature enabled:
+1. Add the following property in your `.csproj`:
+   ```xml
+   <PropertyGroup>
+   		<PublishAot>true</PublishAot>
+	</PropertyGroup>
+   ```
+1. Add the following items in your `.csproj`:
+   ```xml
+   <ItemGroup>
+		<RdXmlFile Include="MyApp.rd.xml" />
+		<RuntimeHostConfigurationOption Include="Switch.System.Reflection.Assembly.SimulatedCallingAssembly" Value="true" />
```

Comment:
That seems to be not needed. At least `SkiaSharpTest.Skia.Gtk` sample works without that, so that's indicating that this is a template problem

Reply:
There are locations in Uno (and the runtime) which use calling assemblies to determine some resources availability, so at this point, it's probably best to keep this available.

Reply:
Because you are vendor, I understand your position. My experience with NativeAOT is that you still have to educate people how to use tweak it to your desired state. So less "magical" tricks the better. Otherwise people will add things which add bloat to app and then complain why things does not working. I do not have nice solutions now, only thing which I can say, is that NativeAOT team would like to make sure that no rd.xml is needed. This one configuration is reasonable, but still, it's hard to change habits for existing customers.

Reply:
Where does Uno use `Assembly.GetCallingAssembly`? We don't use it in the runtime because it's a pretty unreliable API. (This switch is also unsupported and undocumented.)

Reply:
It's not used explicitly in this repo, but if I recall correctly, it is used by `ResourceManager` when resolving resources provided by resx files. It also used in some cases by code using `Type.GetType(string)` without the assembly qualifier in third party libraries.

Reply:
> It's not used explicitly in this repo, but if I recall correctly, it is used by `ResourceManager` when resolving resources provided by resx files. It also used in some cases by code using `Type.GetType(string)` without the assembly qualifier in third party libraries.

The only place in the entire dotnet/runtime repo that calls GetCallingAssembly is in Microsoft.VisualBasic. I really don't think this undocumented switch is needed if Uno doesn't call this API.

Reply:
This assumption may have been based on older mono's implementation :) I'll remove it then. Thanks for the research!